alerts_service/models.py

```diff
@@ -1,29 +1,48 @@
+from datetime import datetime
+from typing import Optional
+
 import sqlalchemy as sa
-from sqlalchemy.orm import declarative_base
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+
 
-Base = declarative_base()
+class Base(DeclarativeBase):
+    pass
```

Comment:
Do we need this?

Reply:
Its something that appears in all the docs so I think good to have